examples/06-Multiphysics/Maxwell3D_Icepak_2Way_Coupling.py

```diff
@@ -0,0 +1,239 @@
+"""
+Multiphysics: Maxwell 3D - Icepak electrothermal analysis
+---------------------------------------------------------
+This example uses PyAEDT to set up a conductor and a ferrite,
+and analyse the resistance increase of the coil because of the
+temperature increase.
+"""
+###########################################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform required imports.
+
+import pyaedt
+from pyaedt.generic.constants import AXIS
```

Comment:
Suggestion: 
It's fine to import constant Axis but the sweep_around_axis also accepts axis name: 
`coil.sweep_around_axis(cs_axis="Z")`
for example :)